cmd/kusk/cmd/deploy.go

```diff
@@ -57,11 +57,15 @@ func init() {
 	//add to root command
 	rootCmd.AddCommand(deployCmd)
 
-	deployCmd.Flags().StringVarP(&file, "file", "f", "", "file to deploy")
+	deployCmd.Flags().StringVarP(&file, "in", "i", "", "file path or URL to OpenAPI spec file to generate mappings from. e.g. --in apispec.yaml")
 	deployCmd.MarkFlagRequired("file")
 
 	deployCmd.Flags().BoolVarP(&watch, "watch", "w", false, "watch file changes and deploy on change")
 	deployCmd.Flags().StringVar(&name, "name", "", "name of the API")
+	deployCmd.Flags().StringVar(&namespace, "namespace", "default", "name of the API")
+	deployCmd.Flags().StringVarP(&envoyFleetName, "envoyfleet.name", "", "kusk-gateway-envoy-fleet", "name of envoyfleet to use for this API. Default: kusk-gateway-envoy-fleet")
+
+	deployCmd.Flags().StringVarP(&envoyFleetNamespace, "envoyfleet.namespace", "", "kusk-system", "namespace of envoyfleet to use for this API. Default: kusk-system")
```

Comment:
another reason that I think it would be good to group this command and others into `api` parent command. We can define these flags as persistent flags once and we can reuse them in child commands

Reply:
We can have it any way we want to. Grouping commands under `api` should be fairly easy. 
@donnalux @aabedraba @crjones can you chime in here